package-res/resources/web/prompting/PromptPanel.js

```diff
@@ -382,6 +382,15 @@ define(['cdf/lib/Base', 'cdf/Logger', 'dojo/number', 'dojo/i18n', 'common-ui/uti
         },
 
         /**
+         * Returns the dashboard store on the prompt panel
+         *
+         * @return {Object}
+         */
+        getDashboard: function() {
+          return this.dashboard;
+        },
+
```

Comment:
If we have getDashboard, probably need re implement showProgressIndicator and hideProgressIndicator from PromptPanel because before that we encapsulated dashboard functions into PromptPanel.